frontend/src/main/pages/HomePage.js

```diff
@@ -25,6 +26,21 @@ export default function HomePage() {
         url: "/api/commons/all"
       }
     );
+  const onSuccess = (commons) => {
+    // Stryker disable next-line all : hard to get variable
+    var existed = new Boolean(false);
+    for(let i = 0; i < commonsJoined.length; i++ ){
+      if(commonsJoined[i].id == commons.id){
```

Comment:
in JS use `===` whenever possible
```suggestion
      if(commonsJoined[i].id === commons.id){
```

Reply:
changed